src/foundry/diceTerms/mathTerm.d.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * A type of RollTerm used to apply a function from the Math library.
+ */
+declare class MathTerm extends RollTerm {
+  /**
+   * @param termData - Data used to create the MathTerm, including the following:
+   *                   (default: `{}`)
+   * @param fn - The named function in the Math environment which should be applied to the term
+   * @param terms - An array of string argument terms for the function
+   */
+  constructor(termData?: Partial<MathTerm.TermData>);
+
+  /** The named function in the Math environment which should be applied to the term */
+  fn: MathTerm.TermData['fn'];
+
+  /** An array of string argument terms for the function */
+  terms: MathTerm.TermData['terms'];
+
+  /** The cached Roll instances for each function argument */
+  rolls: Roll[];
+
+  /** The cached result of evaluating the method arguments */
+  result: number;
+
+  /* -------------------------------------------- */
+  /*  Math Term Attributes                        */
+  /* -------------------------------------------- */
+
+  /** An array of evaluated DiceTerm instances that should be bubbled up to the parent Roll */
+  get dice(): DiceTerm[];
```

Comment:
```suggestion
  get dice(): Roll[];
```

Reply:
Why should this term return an array of rolls? The foundry code states it's an array of `Dice`. I checked with Atropos and it should say DiceTerm.

Reply:
Yeah you are right, I misread the source.